pkg/telemetry/cassandra_agent/cassandra_agent_config.go

```diff
@@ -25,8 +25,7 @@ var (
 	agentConfigLocation = "/opt/management-api/configs/metrics-collector.yaml"
 	defaultAgentConfig  = telemetryapi.CassandraAgentSpec{
 		Endpoint: &telemetryapi.Endpoint{
-			Port:    "9000",
-			Address: "127.0.0.1",
+			Port: "9000",
```

Comment:
Do we even need this? It's port 9000 by default.

Reply:
ok, I've removed it 👍